Documentation/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM docker.io/library/python:3.7.7-alpine3.11
+FROM docker.io/library/python:3.7.9-alpine3.11
```

Comment:
maybe alpine3.12 as well :)

Reply:
Check the PR description ;)

Reply:
:sweat_smile:

My first understanding is alpine 3.12 _alone_ will fail, so just wanna check if python 3.7.9 + alpine 3.12 fails as well ?

Reply:
Using `docker.io/library/python:3.7.9-alpine3.12` produces the following error:
```
Documentation $ make
tar c requirements.txt Dockerfile \
  | docker build --tag cilium/docs-builder -
Sending build context to Docker daemon  10.24kB
Step 1/8 : FROM docker.io/library/python:3.7.9-alpine3.12
 ---> 295b051ee125
Step 2/8 : LABEL maintainer="maintainer@cilium.io"
 ---> Using cache
 ---> 6c47711edd26
Step 3/8 : RUN apk add --no-cache --virtual --update     aspell-en     nodejs     npm     bash     ca-certificates     enchant     git     libc6-compat     py-pip     python     sphinx-python     && true
 ---> Running in ba7a2dd253f1
fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/main/x86_64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/community/x86_64/APKINDEX.tar.gz
ERROR: unsatisfiable constraints:
  enchant (missing):
    required by: --update-20201005.120227[enchant]
  python (missing):
    required by: --update-20201005.120227[python]
  sphinx-python (missing):
    required by: --update-20201005.120227[sphinx-python]
The command '/bin/sh -c apk add --no-cache --virtual --update     aspell-en     nodejs     npm     bash     ca-certificates     enchant     git     libc6-compat     py-pip     python     sphinx-python     && true' returned a non-zero code: 4
make: *** [Makefile:15: builder-image] Error 4
```
With 3.11, no issue.

Reply:
Thanks :100:

Reply:
> With 3.11, no issue.

Why not bump to 3.11 then?

Reply:
@pchaigno I don't understand your comment: it is already using Alpine 3.11. Are you referring to Python? There's no such thing as Python 3.11 (yet).